cmd/login.go

```diff
@@ -61,10 +61,9 @@ func Login() *cobra.Command {
 
 			authorizationURL := buildAuthorizationURL(handler.baseURL, handler.state, port)
 			fmt.Println("Authentication will continue in your default browser")
-			if err := open.Start(authorizationURL); err != nil {
-				fmt.Printf("Please open a browser and navigate to the following address:\n")
-				fmt.Println(authorizationURL)
-			}
+			open.Start(authorizationURL)
+			fmt.Printf("You can also open a browser and navigate to the following address: ")
```

Comment:
I removed the new line. Looks better to me, but we can add that newline if it helps.